charts/prometheus-elasticsearch-exporter/templates/deployment.yaml

```diff
@@ -109,7 +109,7 @@ spec:
                     "--es.snapshots",
                     {{- end }}
                     {{- if .Values.es.cluster_settings }}
-                    "--es.cluster_settings",
+                    "--collector.clustersettings",
```

Comment:
https://github.com/prometheus-community/elasticsearch_exporter/releases/tag/v1.6.0 related to change in 1.6.0